README.md

```
@@ -287,7 +287,7 @@ See the `LICENSE` file for details.
 
 🧠 *Maintained by Coxygen Global – Bernard Sibanda*
 📅 *Last updated: 15 September 2025*
-
+*Developer: Balogun Muiz Dolapo*
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)